x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -131,7 +132,7 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
     options: { stripEmptyFields: false },
     schema,
   });
-  const { getFields, getFormData, reset, submit } = form;
+  const { getFields, getFormData, reset, submit, setFieldValue } = form;
```

Comment:
You shouldn't need `setFieldValue` here, you can instead use `[field].setValue` as in [`handleResetIndices`](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx#L200-L203) below.

Reply:
Thanks, I think this is a dead variable at this point and I don't use it below anymore so this will be removed.